frontends/election-manager/src/screens/unconfigured_screen.tsx

```diff
@@ -65,8 +74,12 @@ export function UnconfiguredScreen(): JSX.Element {
   );
   const [isLoading, setIsLoading] = useState(true);
   const [vxElectionFileIsInvalid, setVxElectionFileIsInvalid] = useState(false);
-  const [client] = useState(new ConverterClient('election'));
-  const [isConvertSems, setIsConvertSems] = useState(false);
+  const [client, setClient] = useState<ConverterClient>();
```

Comment:
does this need to be state? it looks like `client` is just derived from props and then never set again

Reply:
It does for the NH one because the state is stored within the class. We didn't have to for the MS one because the state was actually stored server side.

Reply:
as we discussed, this seems ok for now. might be worth exploring separating out the code that needs the converter into a subcomponent in order to clarify where `client` is actually needed so we don't need to assert whether it's defined or not